apps/docs/content/components/Bento.mdx

```diff
@@ -0,0 +1,227 @@
+| :------------------- | :------------------------------------------------------------------------------------- | :-------: | :------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| `className`          | `string`                                                                               |           | `false`  | Sets a custom class on the root element                                                                                         |
+| `children`           | `React.ReactNode`, `React.ReactNode[]`                                                 |           | `false`  | The children to be rendered within the `Bento.Item`                                                                             |
+| `columnStart`        | `1`, `2`, `3`, `4`, `5`, `6`, `7`, `8`, `9`, `10`, `11`, `12`, `ResponsiveColumnIndex` |           | `false`  | The column number in which this content should start                                                                            |
+| `columnSpan`         | `1`, `2`, `3`, `4`, `5`, `6`, `7`, `8`, `9`, `10`, `11`, `12`, `ResponsiveColumnIndex` |           | `false`  | The number of columns this content should span                                                                                  |
+| `rowStart`           | `number`, `ResponsiveRowIndex`                                                         |           | `false`  | The row number in which this content should start                                                                               |
+| `rowSpan`            | `number`, `ResponsiveRowIndex`                                                         |           | `false`  | The number of rows this content should span                                                                                     |
+| `flow`               | `"row"`, `"column"`, `ResponsiveFlow`                                                  |           | `false`  | The flow of the `Bento.Content` and `Bento.Visual` within the `Bento.Item`                                                      |
+| `colorMode`          | `"light"`, `"dark"`                                                                    | `"light"` | `false`  | The color mode specified for this `Bento.Item`                                                                                  |
+| `visualAsBackground` | `boolean`                                                                              |  `false`  | `false`  | Determines if the `Bento.Visual` child element is set as the background for the `Bento.Item` with the `Bento.Content` overlayed |
+| `styleOrder`         | `"default"`, `"reversed"`, `ResponsiveStyleOrder`                                      | `default` | `false`  | Determines the visual order for the `Bento.Item`. This changes the semantic ordering of elements for accessibility.             |
```

Comment:
Should this prop be called order instead of styleOrder, given it isn't just only a visual change as you mention in the description?